src/app/core/data/request.service.spec.ts

```diff
@@ -198,44 +180,7 @@ describe('RequestService', () => {
       });
     });
 
-    describe(`if the request with the specified UUID wasn't sent, because it was already cached`, () => {
```

Comment:
Is there a reason we removed the test here to verify we are obtaining the cached version of the request?  Is this now tested elsewhere?

Reply:
The behavior that test was verifying has been removed by this PR